src/js/components/projects/detail.tsx

```diff
@@ -202,13 +209,30 @@ const ProjectDetail = (props: RouteComponentProps) => {
   const { branchLink, branchLinkText } = getBranchLink(project);
   const onRenderHeaderActions = () => (
     <PageHeaderControl>
-      <Button
+      <Dropdown
+        align="right"
         iconCategory="utility"
-        iconName="delete"
-        iconPosition="left"
-        label={i18n.t('Delete Project')}
-        variant="text-destructive"
-        disabled
+        iconName="down"
+        iconPosition="right"
+        width="xx-small"
+        className="dropdown-btn"
+        triggerClassName="slds-m-right_xx-small"
+        onSelect={openEditModal}
+        label={
+          <Icon
+            category="utility"
+            name="settings"
+            size="x-small"
+            className="icon-link slds-m-bottom_xxx-small"
+          />
+        }
+        options={[
+          { label: 'Project Options', type: 'header' },
+          { label: 'Edit Name', value: 'edit' },
+          { label: 'Edit Description', value: 'edit' },
+          { type: 'divider' },
+          { label: 'Archive Project', value: 'archive' },
+        ]}
```

Comment:
I propose we combine the two "edit" options, and remove the header.

```suggestion
        options={[
          { label: 'Edit Project' },
          { type: 'divider' },
          { label: 'Delete Project', disabled: true },
        ]}
```